api/v1/peers.go

```diff
@@ -0,0 +1,61 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+)
+
+type Peers struct {
+	Peers []Peer
+}
+
+type peersJSON struct {
+	Data []Peer                 `json:"data"`
+	Meta map[string]interface{} `json:"meta,omitempty"`
```

Comment:
I think that we should not support this field.  It isn't properly supported across beacon nodes, and the only information it provides can be calculated very easily.

Reply:
This would also allow use to return `[]*Peer` rather than the `Peers` struct, which would be more in keeping with the rest of the API.

Reply:
> This would also allow use to return `[]*Peer` rather than the `Peers` struct, which would be more in keeping with the rest of the API.

I like this better, I had been using round trip parsing of responses for the tests so I added it to conform. As its one way I think its best to just remove it.